source/slang/slang.cpp

```diff
@@ -1898,9 +1898,10 @@ struct FrontEndPreprocessorHandler : PreprocessorHandler
     // by applications to decide when they need to "hot reload"
     // their shader code.
     //
-    void handleFileDependency(String const& path) SLANG_OVERRIDE
+    void handleFileDependency(String const& path, ISlangBlob* sourceBlob) SLANG_OVERRIDE
     {
         m_module->addFilePathDependency(path);
+        m_module->getContentsDigestBuilder().addToDigest(sourceBlob);
```

Comment:
What happens if sourceBlob is nullptr?

Reply:
That check slipped my mind when I went to add this, added.